corehq/toggles.py

```diff
@@ -1563,6 +1563,16 @@ def _commtrackify(domain_name, toggle_is_enabled):
     [NAMESPACE_DOMAIN],
 )
 
+SESSION_ENDPOINTS = StaticToggle(
+    'session_endpoints',
+    'Enable session endpoints',
+
+    TAG_SOLUTIONS,
```

Comment:
Know if this should be `TAG_SOLUTIONS_LIMITED`, `OPEN`, or `CONDITIONAL`?

Reply:
I'm not sure. But I bet @karmuno knows.

Reply:
Tagged it "limited use" for now - it's an easy thing to change later as desired.